app/src/main/res/values/strings.xml

```diff
@@ -291,108 +291,109 @@
+    <string name="draft_picker_desc">Long tap → delete.</string>
     <string name="draft_deleted">Draft deleted.</string>
     <string name="dont_crop_media_thumbnail">Don\'t crop media thumbnail (app restart required)</string>
     <string name="custom_stream_listener">Custom notification listener</string>
     <string name="configuration_url">Configuration URL</string>
     <string name="secret">Secret</string>
     <string name="test">Test</string>
-    <string name="custom_stream_listener_desc">This setting is for advanced users. If you are not sure, please don\'t edit.</string>
-    <string name="custom_stream_listener_was_reset">Custom notification listener was reset.</string>
-    <string name="input_url_and_secret_then_test">Input URL and Secret, then tap Test button.</string>
-    <string name="tab_indicator_color">Tab\'s indicator color</string>
+    <string name="custom_stream_listener_desc">Setting for advanced users.</string>
+    <string name="custom_stream_listener_was_reset">Custom notification listener reset.</string>
+    <string name="input_url_and_secret_then_test">Input URL and secret, then tap the \"Test\" button.</string>
+    <string name="tab_indicator_color">Tab indicator colour</string>
     <string name="plugin">Plugin</string>
     <string name="select_plugin">Select plugin</string>
-    <string name="plugin_not_installed">Plugin is not installed.</string>
+    <string name="plugin_not_installed">Plugin not installed.</string>
     <string name="language_code">en</string>
     <string name="recommended_plugin">Recommended plugin</string>
     <string name="tablet_mode">Tablet mode</string>
     <string name="appearance">Appearance</string>
     <string name="performance">Performance</string>
     <string name="behavior">Behavior</string>
-    <string name="notification_on_off_desc">See also account setting to enable/disable notification.</string>
+    <string name="notification_on_off_desc">See also account setting to enable/disable notifications.</string>
     <string name="post">Post</string>
-    <string name="toot_button_default_account">Default account when toot button tapped</string>
-    <string name="account_change_failed_old_draft_has_no_in_reply_to_url">Account change failed. Old draft data has no in_reply_to_url, can\'t convert in_repley_to for selected instance.</string>
+    <string name="toot_button_default_account">Default account when \"Toot\" button is tapped</string>
+    <string name="account_change_failed_old_draft_has_no_in_reply_to_url">Could not switch account. Old draft data has no in_reply_to_url, can\'t convert in_reply_to for selected instance.</string>
     <string name="in_reply_to_id_conversion_failed">\"in_reply_to\" ID conversion failed.</string>
-    <string name="prior_chrome_custom_tabs">Prior Google Chrome (if using Custom Tabs is enabled)</string>
+    <string name="prior_chrome_custom_tabs">Prioritise Google Chrome (\"Custom tabs\" is turned on)</string>

```

Comment:
options to custom tabs is "dont_use_custom_tabs" .  to avoid confusing, use "enabled" instead of "turned on".

Reply:
"Enabled", as opposed to "turned on", means they are ready to be turned on to me, changed.